preview/snap/snapcraft.yaml

```diff
@@ -61,7 +61,14 @@ parts:
         echo $version > $file
       fi
       echo "getting powershell $version"
-      curl -L -o powershell.tar.gz https://github.com/PowerShell/PowerShell/releases/download/v$version/powershell-$version-linux-x64.tar.gz
+      case "$SNAPCRAFT_ARCH_TRIPLET" in
+        aarch64-linux-gnu)
+          curl -L -o powershell.tar.gz https://github.com/PowerShell/PowerShell/releases/download/v$version/powershell-$version-linux-arm64.tar.gz ;;
+        arm-linux-gnueabihf)
+          curl -L -o powershell.tar.gz https://github.com/PowerShell/PowerShell/releases/download/v$version/powershell-$version-linux-arm32.tar.gz ;;
+        *)
```

Comment:
Can we be more explicit here?

Reply:
we can add an explicit `x86_64-linux-gnu` block and then fail if it doesn't match any of the three options..